config/yggdrasil.conf

```diff
@@ -0,0 +1,26 @@
+{
+  "Peers": [
+    "https://corn.chowder.land:443",
+    "tls://44.234.134.124:443",
+    "tcp://celehner.com:12345",
+    "tls://ygg-nv-us.incognet.io:8884"
+  ],
+  "InterfacePeers": {},
+  "Listen": [],
+  "AdminListen": "unix:///var/run/yggdrasil/yggdrasil.sock",
+  "MulticastInterfaces": [
+    {
+      "Regex": ".*",
+      "Beacon": true,
+      "Listen": true,
+      "Port": 0
+    }
+  ],
+  "AllowedPublicKeys": [],
+  "PublicKey": "dd027f4cd4f6419c52342a84ac5f8cee7c3c4b1534872cf6120a1bdc03c592d9",
+  "PrivateKey": "658b0a5f1898451e447e680c267d67cf2f399535c7205faea0179350e4b43830dd027f4cd4f6419c52342a84ac5f8cee7c3c4b1534872cf6120a1bdc03c592d9",
```

Comment:
Is the private key required to be shared?

Reply:
That's a good point, I think I made a mistake by committing the final conf file here instead of generating it.
I don't think yggdrasil will work as I originally hoped inside GitPod docker containers or GitHub codespaces, so I'll put this on the backburner for now.